portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsEntry.testcase

```diff
@@ -40,79 +40,93 @@ definition {
+	@description = "LPS-110392 - Verifies if the user can create a question with an AL image."
 	@priority = 4
 	@refactordone
 	test CanAddQuestionWithDepotImage {
-		JSONDepot.addDepot(
-			depotDescription = "This is the description of a depot",
-			depotName = "Test Depot Name");
-
-		JSONDocument.addFileWithUploadedFile(
-			dmDocumentDescription = "DM Document Description",
-			dmDocumentTitle = "Document_2.jpeg",
-			groupName = "Test Depot Name",
-			mimeType = "image/jpeg",
-			sourceFileName = "Document_2.jpeg");
-
-		JSONDepot.connectSite(
-			depotName = "Test Depot Name",
-			groupName = "Guest");
-
-		JSONMBMessage.addCategory(
-			categoryDescription = "Description",
-			categoryName = "Questions Test Category",
-			userEmailAddress = "test@liferay.com");
+		task ("Given a depot image is Adding to a question and connected a site") {
+			JSONDepot.addDepot(
+				depotDescription = "This is the description of a depot",
+				depotName = "Test Depot Name");
+
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentDescription = "DM Document Description",
+				dmDocumentTitle = "Document_2.jpeg",
+				groupName = "Test Depot Name",
+				mimeType = "image/jpeg",
+				sourceFileName = "Document_2.jpeg");
+
+			JSONDepot.connectSite(
+				depotName = "Test Depot Name",
+				groupName = "Guest");
+		}
 
-		Questions.askQuestionFillingTheMandatoryFields(
-			contentField = "Body AddQuestionWithTheALImage",
-			depotName = "Test Depot Name",
-			imageFileName = "Document_2.jpeg",
-			layoutName = "questions-page",
-			siteURLKey = "guest",
-			titleField = "Title AddQuestionWithTheALImage",
-			topicName = "questions-test-category");
+		task ("And has added a category") {
+			JSONMBMessage.addCategory(
+				categoryDescription = "Description",
+				categoryName = "Questions Test Category",
+				userEmailAddress = "test@liferay.com");
+		}
 
-		Questions.gotoDetails(
-			questionTabField = "Questions",
-			questionTitle = "Title AddQuestionWithTheALImage");
+		task ("When Questions ask a question filling the mandatory fields") {
+			Questions.askQuestionFillingTheMandatoryFields(
+				contentField = "Body AddQuestionWithTheALImage",
+				depotName = "Test Depot Name",
+				imageFileName = "Document_2.jpeg",
+				layoutName = "questions-page",
+				siteURLKey = "guest",
+				titleField = "Title AddQuestionWithTheALImage",
+				topicName = "questions-test-category");
+
+			Questions.gotoDetails(
+				questionTabField = "Questions",
+				questionTitle = "Title AddQuestionWithTheALImage");
+		}
 
-		Questions.viewImageInQuestion(imageFileName = "document_2-jpeg");
+		task ("Then The image associated with the question should be displayed.") {
+			Questions.viewImageInQuestion(imageFileName = "document_2-jpeg");
+		}
 	}
 
-	@description = "Validate if the user can create a question with an image."
+	@description = "LPS-110392 - Verifies if the user can create a question with an image."
```

Comment:
LPS-110392 - Verify if the user can create a question with an image from the DM Library